generators/server/templates/src/main/java/package/config/_JHipsterProperties.java

```diff
@@ -252,7 +252,7 @@ public Authentication getAuthentication() {
         public LoadBalancedResourceDetails getClientAuthorization() {
             return clientAuthorization;
         }
-        <%_ } _ %>
+        <%_ } _%>
```

Comment:
How did you even find this :O 
I was trying to find what was causing it for almost an hour :(

Nice work